### PR TITLE
Allow flash configuration

### LINKED ISSUE #270: Allow disabling flash
**Is your feature request related to a problem? Please describe.**

When I use `vit` and perform an action like "start a task" and "stop a task" the task I'm working on flashes.

I believe the internal function used for this is `flash_focus`.

**Describe the solution you'd like**

Ideally, I would like to disable flashing completely via config option or similar.

**Additional context**

I'd like `vit` to be as fast as possible so I don't like waiting for flashes.

### PR DESCRIPTION
**Pull request by JKubovy**
Add 'flash_focus_repeat_times' and 'flash_focus_pause_seconds' keys to [vit] section in config.ini. Also support int and float types in config and automatically parse (transform) them. Keep same default values.

fixes vit-project/vit#270

### CODE REVIEW

File: vit/config_parser.py [hunk 1]
```diff
@@ -166,7 +168,7 @@ def get(self, section, key):
         try:
             value = self.config.get(section, key)
             return self.transform(key, value, default)
-        except (configparser.NoSectionError, configparser.NoOptionError):
+        except (configparser.NoSectionError, configparser.NoOptionError, ValueError):
```
Comment (thehunmonkgroup) [inline vit/config_parser.py:171]: Why did you add `ValueError` here?

Comment (JKubovy) [reply vit/config_parser.py:171]: Because if you parse int or float you can get ValueError. In that case you need to use default value.

**Comment by thehunmonkgroup** [comment]
Works great, thanks for your efforts!

**Comment by scottkosty** [comment]
@JKubovy Thanks! Just curious, which values do you choose for the new configs? (edit: ah, after looking at the issue I am guessing you use `flash_focus_repeat_times=0`)

**Comment by thehunmonkgroup** [comment]
In my own configuration? The defaults, I'm happy with the way it works now.
